applications/nrf5340_audio/src/nrf5340_audio_common.c

```diff
@@ -4,8 +4,12 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
+#include "audio/audio_system.h"
 #include "bt_mgmt.h"
+#include "channel_assignment.h"
 #include "fw_info_app.h"
+#include "nrfx_clock.h"
```

Comment:
```suggestion
#include <nrfx_clock.h>
```
and then move up to differentiate "our" header files vs system header files